src/xform_to_datalog/testdata/ok_claim_propagates.dl

```diff
@@ -2,6 +2,7 @@
 
 #include "taint.dl"
 #include "may_will.dl"
+#include "auth_logic_type_interface.dl"
```

Comment:
There are a number of `BUILD` files in the project where we list included `dl` files for certain tests. Even if the build system is managing to locate this file without it being explicitly named there, we should probably add it in those places so that the build system correctly re-triggers builds if this file were to change. I believe you will want to update the `BUILD` file under `arcs_fact_tests`, maybe others.